app/models/manageiq/providers/autosde/inventory/collector/storage_manager.rb

```diff
@@ -73,4 +73,17 @@ def physical_storage_families
       }
     end
   end
+
+  def storage_hosts_wwpn_candidates
+    @storage_hosts_wwpn_candidates ||= @manager.autosde_client.StorageHostWWPNCandidatesApi
+                                               .storage_hosts_wwpn_candidates_get.map do |candidate|
+      {
+        :candidates => candidate.wwpn,
```

Comment:
Minor formatting issue but can you line up the ` =>` with the hash rockets below?
E.g.
```ruby
{
  :candidates          => candidate.wwpn,
  :physical_storage_id => candidate.system_uuid,
  :ems_ref             => candidate_system_uuid
}
```